packages/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.17+1
```

Comment:
Can you change this to `0.5.19` please. Thanks

Reply:
Done